local.tf

```diff
@@ -0,0 +1,26 @@
+locals {
```

Comment:
move this out to aks-build-infrastructure, probably in a module so you don't have to duplicate it?